# This msg file specifies a sequence of 3D poses

pybullet_motoman/Pose3D[] pose_sequence
